spec/acceptance/throttling_spec.rb

```diff
@@ -34,6 +34,27 @@
     end
   end
 
+  it "returns correct Retry-After header if enabled" do
+    begin
+      prev = Rack::Attack.retry_after_header_enabled
+      Rack::Attack.retry_after_header_enabled = true
```

Comment:
I know I suggested this name in https://github.com/kickstarter/rack-attack/issues/209#issuecomment-398841599, but giving it a second thought, should we name it `Rack::Attack.throttled_response_retry_after_header = true` so that it's clear to the user that the setting is scoped to the throttled response only and not something else?